crates/ruff_python_formatter/tests/snapshots/black_compatibility@simple_cases__multiline_consecutive_open_parentheses_ignore.py.snap

```diff
@@ -33,18 +33,19 @@ print(   "111"       ) # type: ignore
 ```diff
 --- Black
 +++ Ruff
-@@ -1,10 +1,8 @@
+@@ -1,12 +1,6 @@
  # This is a regression test. Issue #3737
  
 -a = (  # type: ignore
-+a = int(  # type: ignore  # type: ignore
-     int(  # type: ignore
+-    int(  # type: ignore
 -        int(  # type: ignore
 -            int(6)  # type: ignore
 -        )
-+        int(6)  # type: ignore
-     )
- )
+-    )
+-)
++a = int(int(int(6)))  # type: ignore  # type: ignore  # type: ignore  # type: ignore
```

Comment:
IMO: The old formatting expressed the intent of the user better.

We, so far, have taken the stand that trailing comments should remain as closely as possible to what they originally commented (see [internal notion](https://www.notion.so/astral-sh/Black-Deviations-65977bec23b64aee9d28cc1787badfdb?pvs=4#b346e0ba9cd84f8e93ed0207e5912048)). Meaning they should, by default, expand their parent. I think this should also help that ignore comments move to far away, changing what they suppress.

Reply:
I will look into how we can preserve this while still fixing the bug in the PR summary, but FWIW, Black also does this formatting, _unless_ the comments are type ignores: https://black.vercel.app/?version=main&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AFIAH1dAD2IimZxl1N_WlOfrjryFgvD4ScVsKPztqdHDGJUg5knO0JCdpUeAEufiQmt7zAZacPYBA_60Y2o5_0gUa0qrRrUebiwy-1h9dFYjpJKI3bTmqTD_atbbMhmkqzeioikfpq_PQL5C6EgvdfH6suhrNVSpATyrIAX7v50BhYnAAAAAGQFt7kMk4ZRAAGZAckCAAAEz53MscRn-wIAAAAABFla.

Reply:
(I think this should be investigated separately from this PR.)

Reply:
Will put up a separate PR.

Reply:
One feature that Ruff's architecture enables is that we can move comments. This is much harder in Black. I think we should use it and be opinionated about where comments should be placed and how they get formatted.

Reply:
Follow-up here: https://github.com/astral-sh/ruff/pull/6389